docs/building-a-form/README.md

```diff
@@ -1,14 +1,16 @@
 # Building a form
 
-### About the JSON schema standard and react-jsonschema-form
+### [About the schemaform](about-the-json-schema-standard-and-react-jsonschema-form.md)
 
-### Adding widgets
+Learn about the form building code, or *schemaform*, and the library it's built on.
```

Comment:
I would rewrite this to be "Learn about how us-forms-system works, and the library it's built on".

Reply:
To give this a little more context, I went with "The US Forms System lets you build web-based forms using React and the JSON Schema standard."